### ISSUE TITLE
Accordion block unable to move or toggle between HTML and visual editor

### ISSUE THREAD

**Issue by gbueno-f1** [forumone/f1-block-library#37]
The following issues were reported by Appalchian Regional Commission in regard to the accordion block in the block library. Ticket: [ARC19-443](https://forumone.atlassian.net/browse/ARC19-443).

**Unable to Move Up/Down** - Once we create an accordion, we can’t move it up or down. The arrows remain gray/unclickable. Here’s a screenshot: [https://snipboard.io/Z5W6gq.jpg](https://snipboard.io/Z5W6gq.jpg)

**Unable to toggle between HTML/Visual Editor** - If we try to edit an accordion via HTML, we cannot return back to the visual editor. We get prompted to attempt block recovery, but the block never recovers. Here’s a screenshot of the error message: [https://snipboard.io/Xme61U.jpg](https://snipboard.io/Xme61U.jpg)

I was able to reproduce both errors.

I've already advised the client to use the Detail block in core, and we've done work to make sure the theming of the accordion matches the detail block.

[ARC19-443]: https://forumone.atlassian.net/browse/ARC19-443?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by timnolte** [comment]
Yeah, so people can't use HTML mode on any blocks, that will break blocks with that error no matter what sort of block it is.

**Comment by timnolte** [comment]
Also, the accordion block is more than likely going to be removed from the F1 Block Library since WordPress Core includes the Details Block now.
